src/components/page-intro.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { formatLargeNumber } from '../utils';
+import '../styles/components/page-intro.scss';
+import ChevronDown from '../svg/chevron-down.svg';
+import ChevronUp from '../svg/chevron-up.svg';
+
+class PageIntro extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showContent: false,
+    };
+  }
+
+  render() {
+    const {
+      title, resultsCount, children, links,
+    } = this.props;
+    const { showContent } = this.state;
+    return (
+      <div className="page-intro">
+        <h2>
+          <button
+            type="button"
+            onClick={() => this.setState({ showContent: !showContent })}
+            className="dropdown"
+          >
+            {showContent ? <ChevronUp /> : <ChevronDown />}
+            {' '}
+            {title}
+          </button>
+          {resultsCount > 0 && (
```

Comment:
I would suggest to put `resultsCount > 0` in a pair of parenthesis, so the conditional statement would be more obvious e.g. `(resultsCount > 0) && ...`

Reply:
Prettier removes it with the current config. That's something we could discuss when we talk about the code styling decisions.

Reply:
It took me a second to realise what does 'Prettier' mean here! Let's chat about it later then!